src/net.cpp

```diff
@@ -1557,7 +1557,7 @@ void CConnman::SocketHandler()
             if (nBytes > 0)
             {
                 bool notify = false;
-                if (!pnode->ReceiveMsgBytes(Span<const uint8_t>(pchBuf, nBytes), notify))
+                if (!pnode->ReceiveMsgBytes({pchBuf, (size_t)nBytes}, notify))
```

Comment:
nit in the second commit, if you retouch:

Add `{}` to the if body?

Reply:
Done.